### PR TITLE
chore(web): asset internationalization

### PR DESCRIPTION
**Pull request by nourbalaha**
# Overview
Added internationalization to asset using i18n.

# What I have done
- Added internationalization to assetList using i18n.
- Added internationalization to asset using i18n.

### CODE REVIEW

File: web/src/components/organisms/Asset/AssetList/hooks.ts [hunk 1]
```diff
@@ -93,17 +95,17 @@ export default (projectId?: string) => {
             });
             if (result.errors || !result.data?.createAsset) {
               // TODO: notification
-              console.log("Failed to add one or more assets.");
+              console.log(t("Failed to add one or more assets."));
```
Comment (issmail-basel) [inline web/src/components/organisms/Asset/AssetList/hooks.ts:98]: @nourbalaha , remove the logging code `console.log`, will replace it with notifications.

File: web/src/components/organisms/Asset/AssetList/hooks.ts [hunk 2]
```diff
@@ -93,17 +95,17 @@ export default (projectId?: string) => {
...
             }
           }),
         );
         if (results) {
           // TODO: notification
-          console.log("Successfully added one or more assets.");
+          console.log(t("Successfully added one or more assets."));
```
Comment (issmail-basel) [inline web/src/components/organisms/Asset/AssetList/hooks.ts:104]: @nourbalaha , remove the logging code `console.log`, will replace it with notifications.

File: web/src/i18n/translations/ja.yml [hunk 3]
```diff
@@ -1,3 +1,33 @@
+Download: ''
```
Comment (issmail-basel) [inline web/src/i18n/translations/ja.yml:1]: @nourbalaha , ask for translation before merging.

**Review by issmail-basel — CHANGES_REQUESTED** [review]
A couple of comments

File: web/src/components/organisms/Asset/AssetList/hooks.ts (hunk 1 above)
Comment (nourbalaha) [reply web/src/components/organisms/Asset/AssetList/hooks.ts:98]: Can you explain more please, do you mean that I should remove the `console.log` line? replace it with what ??
As far as I know, we haven't implemented notifications in reearth-cms, no?

File: web/src/components/organisms/Asset/AssetList/hooks.ts (hunk 2 above)
Comment (nourbalaha) [reply web/src/components/organisms/Asset/AssetList/hooks.ts:104]: Can you explain more please, do you mean that I should remove the console.log line? replace it with what ??
As far as I know, we haven't implemented notifications in reearth-cms, no?

File: web/src/i18n/translations/ja.yml (hunk 3 above)
Comment (nourbalaha) [reply web/src/i18n/translations/ja.yml:1]: I will ask Baba-san to translate once you and Kyle approve this PR.

File: web/src/components/organisms/Asset/AssetList/hooks.ts (hunk 2 above)
Comment (issmail-basel) [reply web/src/components/organisms/Asset/AssetList/hooks.ts:104]: Yes, remove `console.log` no need for it here, it's for debugging purposes only.

**Review by issmail-basel — APPROVED** [review]
Remove `console.log`, other than that LGTM.

File: web/src/components/molecules/Asset/AssetList/AssetListTable/index.tsx [hunk 4]
```diff
@@ -53,42 +55,42 @@ const AssetListTable: React.FC<AssetListTableProps> = ({
...
     },
     {
-      title: "Created By",
+      title: t("Created By"),
       dataIndex: "createdBy",
       key: "createdBy",
     },
     {
-      title: "Id",
+      title: t("Id"),
```
Comment (KaWaite) [inline web/src/components/molecules/Asset/AssetList/AssetListTable/index.tsx:88]: ID

Comment (KaWaite) [inline web/src/components/molecules/Asset/AssetList/AssetListTable/index.tsx:88]: title: t("ID")

**Review by KaWaite — APPROVED** [review]
Please update the Id -> ID and then good to go!

File: web/src/components/molecules/Asset/AssetList/AssetListTable/index.tsx (hunk 4 above)
Comment (nourbalaha) [reply web/src/components/molecules/Asset/AssetList/AssetListTable/index.tsx]: Done!

**Pull request merged**